setup_vs_tools.cmd

```diff
@@ -0,0 +1,44 @@
+@if not defined _echo @echo off
```

Comment:
The documentation for this script should say what it expects on input and what it sets on output.